src/components/dialogs/parameters/map-parameters.jsx

```diff
@@ -4,25 +4,27 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-
+//TODO: File in JSX because @powsybl/diagram-viewer doesn't provide type. change to TSX when types are awailable.
```

Comment:
Can be done with v0.5.4 of diagram-viewer merged in #2177

Reply:
done